### ISSUE TITLE
Unable to connect to graphic server

### ISSUE THREAD

**Issue by Auerbath** [joshpatten/PVE-VDIClient#37]
Hi there! I'm trying to get everything work by this [video](https://www.youtube.com/watch?v=oLatrZBFQrw) 
I have configured VDIClient, as well as user and VM configuration in proxmox.
So the problem I have can be seen on screenshot 
![1](https://user-images.githubusercontent.com/105651963/201094379-8c0068a2-a591-464d-b69d-c2ae0b606e1e.png)

I have no problem connecting to my proxmox, but after I choose the VM I want to connect, I tried 2 systems (Ubuntu 22 with disabled wayland, debian11lxde) - it's giving me an error "Unable to connect to graphic server" referring to cloudflare IP address.

Any ideas what could cause this issue, please?
Maybe someone has had this problem before, would appreciate the help
Any additional info - feel free to ask

**Comment by doughnet** [comment]
Same issue for me.  Just tested it out with a Windows 2019 server VM.

VNC and SPICE works through Proxmox Web

**Comment by doughnet** [comment]
No difference.  Not sure why one PVE works and another doesn't.

**Comment by doughnet** [comment]
I figured out my issue.  The PVE that worked has a hostname that is accessible via WAN (i.e. pve.domain.com).  The PVE that did not work an internal hostname which does not resolve via WAN/outside the local networking (i.e. pve.home.local).

By utilizing the SpiceProxyRedirect flags within the vdiclient.ini file I was able to make a redirect like below:
**[SpiceProxyRedirect]
pve.home.local:3128 = 192.168.1.60:3128**
(what the hostname is):(default port) = (IP.Address):(default port)

Hope this helps solve an issue for someone else.
